routing/search.js

```diff
@@ -0,0 +1,162 @@
+'use strict';
+
+const express = require('express');
+const router = express.Router();
+// const fs = require('fs');
+// const path = require('path');
+// const dotenv = require('dotenv');
+const dbreader = require('../db/dbreader');
+
+const dbconfig = {
+  connectionString: process.env.DATABASE_URL
+};
+
+function parseBooks(rows) {
+  let arr = '';
+
+  /*rows.forEach(row => {
+    arr += '<div class=\'test\'><img class=\'cover\' src=\'' + row.preview + '\'><p class=\'description\'>' +
```

Comment:
build your img src by:
[img src="https://${process.env.BUCKET}.s3.us-east-2.amazonaws.com/${row.photos[0]}" class="..."]
// in output you will get url like this: https://ubiquitous-spark.s3.us-east-2.amazonaws.com/photos/js.png